api/agent/instancemutater/machine_test.go

```diff
@@ -169,7 +169,7 @@ func (s *instanceMutaterMachineSuite) TestCharmProfilingInfoSuccessChangesWithNo
 	defer s.setup(c).Finish()
 
 	args := params.Entity{Tag: s.tag.String()}
-	results := params.CharmProfilingInfoResult{
+	results := params.CharmProfilingInfoResultV4{
```

Comment:
The caller needs to be aware of what version is being used on the controller and make the necessary logic changes to adapt. Use `m.facade.BestAPIVersion()`